app/Repositories/Serializers/FeaturedTourSerializer.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Repositories\Serializers;
+
+use App\Models\Transformers\FeaturedTourTransformer;
+use League\Fractal\Manager;
+use League\Fractal\Resource;
+
+class FeaturedTourSerializer
+{
+    protected ?Manager $manager = null;
```

Comment:
@zachgarwood what does adding a ? here do?

Reply:
Oh, it just indicates that the variable type could also be `null`. It's equivalent to `Manager|null`.

Here's the reference: https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.nullable